prow/git/v2/adapter.go

```diff
@@ -49,7 +59,7 @@ func (a *repoClientAdapter) Commit(title, body string) error {
 }
 
 func (a *repoClientAdapter) ForcePush(branch string) error {
-	return errors.New("no ForcePush implementation exists in the v1 repo client")
+	return a.Repo.Push(branch)
```

Comment:
So while I am okay changing the cherrypicker to force push, having a force push that does not force push is IMHO not that great. Maybe just add it to the v1 client/replace the push if that was the only occurernce of irtbeing used?

Reply:
There is no change in behavior here -- the push was always a force

Reply:
I just called that out in the method name